hashintel/hash
# Don't open HASH internal URLs in new tab

## 🌟 What is the purpose of this PR?




This PR treats all links that have `hash.dev`, `hash.ai` or `blockprotocol.org` as their top-level domain as "internal" URLs (so that they aren't opened in a new tab)

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1202176479837585/1202286528190283/f) _(internal)_

## 🚫 Blocked by





## 🔍 What does this change?




- updates the `isHrefExternal` method to account for HASH domains that are considered "internal"

## 📜 Does this require a change to the docs?



No.

## 🛡 What tests cover this?



-

## ❓ How to test this?



1. Open the deployment
2. Click on any link that re-directs to `hash.ai`, `hash.dev` or `blockprotocol.org` to verify they aren't opened in new tabs
